test/functional/connection/connection.ts

```diff
@@ -359,5 +359,33 @@ describe("Connection", () => {
         })));
 
     });
+
+    describe("Can change postgres default schema name", () => {
+        let connections: Connection[];
+        beforeEach(async () => {
+            connections = await createTestingConnections({ 
+                enabledDrivers: ["postgres"],
+                entities: [Post]
```

Comment:
you just need to send `schemaName` just like `entities` are. You probably need to modify `test-utils`.